colossalai/booster/plugin/low_level_zero_plugin.py

```diff
@@ -346,7 +317,7 @@ def configure(
                 optimizer, **self.zero_optim_kwargs, verbose=self.verbose
             )
             # inject update_master_params
-            model.update_master_params = MethodType(optimizer.update_master_params, model)
+            model.module.update_master_params = MethodType(optimizer.update_master_params, model)
```

Comment:
I think we'd better unify the behavior of `unwrap()`. The only effect of `unwrap()` is to get the original model before boosting. General checkpoint io should not unwrap model by default. Whether to unwrap model should be controlled by subclass. And this line should not be changed.

Reply:
modified